src/coreComponents/unitTests/meshTests/testVTKImport.cpp

```diff
@@ -115,31 +115,28 @@ TEST( VTKImport, cube )
 
     // 1 elements type on 3 regions ("-1", "3", "9") = 3 sub-groups
     ASSERT_EQ( cellBlockManager.getCellBlocks().numSubGroups(), 3 );
-    GEOSX_LOG_RANK( cellBlockManager.getCellBlocks().getGroup( 0 ).getName() );
-    GEOSX_LOG_RANK( cellBlockManager.getCellBlocks().getGroup( 1 ).getName() );
-    GEOSX_LOG_RANK( cellBlockManager.getCellBlocks().getGroup( 2 ).getName() );
 
     // FIXME How to get the CellBlock as a function of the region, without knowing the naming pattern.
-    CellBlockABC const & hexs = cellBlockManager.getCellBlocks().getGroup< CellBlockABC >( "hexahedra" );
-    CellBlockABC const & hexs3 = cellBlockManager.getCellBlocks().getGroup< CellBlockABC >( "3_hexahedra" );
-    CellBlockABC const & hexs9 = cellBlockManager.getCellBlocks().getGroup< CellBlockABC >( "9_hexahedra" );
+    std::array< std::pair< string, int >, 3 > const expectedCellBlocks =
```

Comment:
Hmmm, these double curly brackets are quite sad.

Reply:
To be honest, I didn't understand why so many curlies were needed. I thought `{ { a, b }, { c, d } }` is sufficient to aggregate-initialize an array of pairs, but the compiler disagreed.